### ISSUE TITLE
Proper Benchmark designing on several survival analysis tasks

### ISSUE THREAD

**Issue by steletvinicius** [mlr-org/mlr3benchmark#47]
Hello, and congratulations on mlr3verse's elegant and efficient interface.
My team and I decided to invest some time on learning its particular notation as it seems to be promissing and really useful.

I'm pursuing my PhD and have been conducting experiments with machine learning and survival analysis on a very large dataset.

I've already familiarized myself with some of the mlr3verse notation and would like to simplify things by using the benchmark function to run all the tests on both the overall dataset and its segments.

I want to test 13 possible ML algorithms using Cox ph as my performance comparison algorithm.

For each task and model, I would like to:

A) on the training set:
1. Find the best hyperparameter configuration using cross-validation, using the Concordance index as a performance metric;
2. Train the final model;

B) On the test set:
1. Measure the performance of all models in not only one but 30 resamples with replacement (bootstraping), using the C-index and other survival metrics as a comparative metrics. 

I would appreciate help on how to write this experiment using the proper benchmark function notation.

Here is part of my script showing what I have done so far:

[https://gist.github.com/steletvinicius/300cfc81a1e85e147335be0f5ad4da0c](url)

Some questions:
Does benchmark function uses only the train partition to run the hyperparameter optimization?
How to access for each task the best fitted model performance on the  training and the test sets?
How to extract the best models fromt he benchmark for future steps, like feature importance evaluations and to release on the production environment?
